tsdoc/src/parser/TSDocMessageId.ts

```diff
@@ -411,7 +416,12 @@ export enum TSDocMessageId {
   /**
    * The code span is missing its closing backtick.
    */
-  CodeSpanMissingDelimiter = 'tsdoc-code-span-missing-delimiter'
+  CodeSpanMissingDelimiter = 'tsdoc-code-span-missing-delimiter',
+
+  /**
+   * The XML start and end tag names are not the same.
+   */
+  XmlTagNameMismatch = 'tsdoc-xml-tag-name-mismatch'
```

Comment:
Is this used? The error message that I see is `Expecting closing tag name "x" to match opening tag name "y"`

Reply:
Looks like the tag is used but the comment is outdated